examples/grafana/README.md

```diff
@@ -0,0 +1,31 @@
+# grafana-ocp
+
+This template creates a custom Grafana instance preconfigured to gather Prometheus openshift metrics.
+It is uses OAuth token to login openshift Prometheus.
+
+
+## To deploy grafana
+Note: make sure to have openshift prometheus deployed.
+(https://github.com/openshift/origin/tree/master/examples/prometheus)
+
+1. ```oc create project grafana```
```

Comment:
it is oc create namespace

oc create project does not work

Reply:
should be:
`oc new-project grafana`

Reply:
yes 👍 create namespace is nicer :-)
`oc create namespace grafana`

Reply:
Will fix that. 10X